### DISCUSSION TITLE
1.21.0

### DISCUSSION THREAD

**Discussion by Mbrick2** [ScratchAddons/ScratchAddons#3637] [Help]
When is the update coming as a stable version???

**Comment by Endermaster3690** [comment]
chrome web store reviewers need to review each update before they come out

**Reply by Mbrick2** [reply]
I mean the version in the manifest on github

**Reply by Endermaster3690** [reply]
oh ok

**Comment by apple502j** [comment]
Monday or Tuesday prob
